### DISCUSSION TITLE
Will fixed buffer be implemented in C#12?

### DISCUSSION THREAD

**Discussion by ygc369** [dotnet/csharplang#6728] [General] [answered]
People have wanted it for quite a long time, and it seems not so hard. I wonder why it has not been added into C# yet. Will it have a chance to be part of C#12?

**Comment by HaloFour** [comment, marked as answer]
See: #1314

**Comment by ygc369** [comment]
I know there is such a champion #1314, but what I want to know is whether this feature is planned in C#12.

**Reply by 333fred** [reply]
There is no guarantee that any feature will come to any specific language version.
